src/Components/AlbumCard/assets/favoriteIcon.svg

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Just me being curious, would we rather import this favorite Icon or the simple heart provided by Material UI?

Reply:
Yes